### ISSUE TITLE
Question about bench2drive_base_train_val_split.json

### ISSUE THREAD

**Issue by weimengchuan** [Thinklab-SJTU/Bench2DriveZoo#10]
Hi, thanks for your great dataset!

I have a question, I found that in in `bench2drive_base_train_val_split.json`, length of train clips are 940, length of val clips are 50. But there are a total of 1000 clips in Bench2Drive-base dataset. 

Why not remaining 10 clips are not listed?
I guess they are not used as test set. Because their scenarios are the same.

The remaining 10 clips are: 
  InterurbanActorFlow_Town06_Route315_Weather3
  InterurbanActorFlow_Town12_Route1292_Weather2
  InterurbanActorFlow_Town13_Route707_Weather5
  InterurbanActorFlow_Town06_Route316_Weather3
  InterurbanActorFlow_Town12_Route843_Weather11
  InterurbanActorFlow_Town13_Route623_Weather25
  InterurbanActorFlow_Town06_Route314_Weather2
  InterurbanActorFlow_Town12_Route938_Weather2
  InterurbanActorFlow_Town06_Route294_Weather8
  InterurbanActorFlow_Town12_Route842_Weather10

Thanks in advance!

**Comment by jiaxiaosong1002** [comment]
@weimengchuan Hi, thanks for pointing out!  It seems that our generation code for the open sourced train-val split json is running on a wrong directory. We will change the preprocess code to the version of our private repo, which only considers val route and the rest of routes under the  v1 directory are all considered as training data. Please stay tuned.

**Comment by jiaxiaosong1002** [comment]
@weimengchuan Hi, now it should work properly.

**Comment by weimengchuan** [comment]
Thanks a lot!

**Comment by weimengchuan** [comment]
Solved.
